lib/puppet/provider/oneview_hypervisor_cluster_profile/c7000.rb

```diff
@@ -24,6 +24,51 @@
 
   mk_resource_methods
 
+  def exists?
+    create
+  end
+
+  def create
+    set_uri
+    @data = resource['data']
+    return true if resource_update
+    super(:create)
+  end
+
+  def set_uri
+    hm_name = resource['data']['hypervisorManagerUri']
+    hm_uri = OneviewSDK.resource_named(:HypervisorManager, api_version).find_by(@client, name: hm_name).first['uri']
+    resource['data']['hypervisorManagerUri'] = hm_uri if resource['data']['hypervisorManagerUri'].empty?
```

Comment:
Will this if condition satisfies as per the above code..?

Reply:
Didnt copy latest change from backup file after testing and corrected it now.